packages/cli/src/commands/build.ts

```diff
@@ -176,10 +176,17 @@ export default async function main(client: Client): Promise<number> {
   if (pkg instanceof CantParseJSONFile) throw pkg;
   if (vercelConfig instanceof CantParseJSONFile) throw vercelConfig;
 
+  const isWin = process.platform === 'win32';
+
   // Get a list of source files
-  const files = (await getFiles(workPath, client)).map(f =>
-    relative(workPath, f)
-  );
+  const files = (await getFiles(workPath, client)).map(f => {
+    let rel = relative(workPath, f);
+    if (isWin) {
+      // On Windows, normalize forward slashes to backslashes
+      rel = rel.replace(/\\/g, '/');
```

Comment:
We have a shared function for this in `@vercel/build-utils`. Can we reuse it here?

Reply:
e9bc16831f1c7837d57ad7647f3d5a2193c26a0b